index.html

```diff
@@ -17,24 +17,24 @@ <h1>Біографія</h1>
     </ul>
     <h2>Улюблені фільми</h2>
     <a href="http://gidonline.in/film/ochen-strashnoe-kino-5/" target="_blank">
-        <img src="images/film1.jpg" alt="film" width="200" height="284,5">
+        <img src="images/film1.jpg" alt="film" width="200" height="284">
```

Comment:
since you already added `width` and `height` attributes into `style.css` file, you should remove these attributes form the `img` tag